experimental/cloudstrapper/playbooks/roles/control/templates/main.tf

```diff
@@ -0,0 +1,89 @@
+################################################################################
```

Comment:
Do we need this file. I think just the templated versions main.tf should do

Reply:
Agree. File removed.